package.json

```diff
@@ -22,21 +23,16 @@
     "url": "https://github.com/RampNetwork/crypto-address-validator"
   },
   "scripts": {
-    "minify": "uglifyjs -c -m -o dist/wallet-address-validator.min.js -- dist/wallet-address-validator.js",
     "test:node": "mocha -r ts-node/register test/*.ts",
     "test:browser": "karma start",
     "test": "yarn run build && npm run test:node && npm run test:browser",
     "test:nobuild": "npm run test:node && npm run test:browser",
     "start": "npm run build && npm test",
     "lint": "eslint --ext .ts .",
     "lint:fix": "yarn lint --fix",
-    "generateReadme": "node debug/generateReadme.js && git add README.md dist/wallet-address-validator.min.js.gz package.json",
-    "createMin": "yarn run bundle && git add dist/wallet-address-validator.js dist/wallet-address-validator.min.js",
-    "preversion": "yarn run createMin && yarn run generateReadme",
-    "postversion": "git push origin HEAD --follow-tags && hub pull-request -o -b dev -m v%npm_package_version%",
     "build:ts": "tsc",
     "build:rollup": "rollup -c rollup.config.ts",
-    "build": "yarn run build:ts && yarn run build:rollup"
+    "build": "rm -rf dist && yarn run build:rollup"
```

Comment:
You should probably use the `rimraf` pkg, because this will fail on windows machines.